index.html

```diff
@@ -14,52 +14,46 @@
 
 <body>
     <header>
-        <div class="gutter-right logo">
+        <!--<div class="gutter-right logo">
             <img src="./assets/big-logo.png" />
-        </div>
+        </div>-->
     </header>
     <main>
         <section class="main-section">
-            <img class="main-logo" src="./assets/alchemy-logo.png" />
+            <!--<img class="main-logo" src="./assets/alchemy-logo.png" />-->
             <h2>
-                Welcome to Alchemy Bootstrap!
+                The magic forest is on fire! The leprechaun are rushing to save their gold! 
             </h2>
-
-            <p>
-                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Aperiam repellendus similique consectetur
-                voluptas facilis molestiae voluptates nobis error sapiente, ad, ipsa vero non, sint eligendi? Eos
-                praesentium iusto magnam exercitationem.
-                Earum sunt voluptatibus molestias, sint voluptatum quos reiciendis. Id asperiores deleniti dolores
-                provident, eius, nostrum aperiam vel, optio qui illo corrupti in. Eum quam, est nemo porro sapiente
-                excepturi assumenda.
-                Rerum unde quidem maxime. Ullam, dolores labore molestias voluptatibus adipisci quod? Quaerat nihil
-                inventore molestiae assumenda, voluptates vel necessitatibus eligendi? Accusamus ratione eligendi
-                reprehenderit? Beatae incidunt molestias ea soluta nulla.
-                Fugiat quae soluta voluptate mollitia praesentium modi ex illo, autem assumenda, vel aliquam fugit
-                possimus dolorem esse corporis ipsa! Voluptatem sapiente error non, ea optio suscipit quasi maxime nobis
-                ex?
-                Et ipsa tenetur ullam ab nesciunt quas fugit velit impedit illum esse eligendi corporis culpa
-                praesentium aspernatur ex sint veniam quidem, doloribus corrupti hic vitae. Eum neque ducimus omnis
-                exercitationem?
-            </p>
+            <div id="test" class="hidingit adg-places">
+                
+                <div id="shell-one" class="hiding-place">
```

Comment:
Great job on this one! Try to use some more semantic HTML elements next time since those will help to make your code even easier to read and be more accessible to users. You can reference the article here for more info: 

[HTML Semantic Elements](https://www.w3schools.com/html/html5_semantic_elements.asp)